test/e2e/flask/multichain-api/testHelpers.ts

```diff
@@ -33,19 +33,24 @@ export const DEFAULT_MULTICHAIN_TEST_DAPP_FIXTURE_OPTIONS = {
   localNodeOptions: [
     {
       type: 'anvil',
+      options: {
+        hardfork: 'muirGlacier',
+      },
     },
     {
       type: 'anvil',
       options: {
         port: 8546,
         chainId: 1338,
+        hardfork: 'muirGlacier',
       },
     },
     {
       type: 'anvil',
       options: {
         port: 7777,
         chainId: 1000,
+        hardfork: 'muirGlacier',
       },
     },
   ],
```

Comment:
these tests relied on metrics from muirGlacier (ie gas values). Open a separate ticket for updating that as it got convoluted in this same PR -- to discuss with multichain team